packages/uploadthing/src/internal/types.ts

```
@@ -92,10 +87,9 @@ type MiddlewareFn<
   },
 ) => MaybePromise<TOutput>;
 
-type ResolverFn<
-  TOutput extends JsonObject | void,
-  TParams extends AnyParams,
-> = (opts: ResolverOptions<TParams>) => MaybePromise<TOutput>;
+type UploadCompleteFn<TMetadata, TOutput extends JsonObject | void> = (
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)